src/foremast/elb/create_elb.py

```diff
@@ -208,3 +221,68 @@ def add_stickiness(self):
                                                                      PolicyName=policyname)
                     stickiness_dict[externalport] = policyname
         return stickiness_dict
+
+
+    def configure_load_balancer_attributes(self, json_data):
+        """Configure load balancer attributes such as idle timeout, connection draining, etc
+
+        Args:
+            json_data (json): return data from ELB upsert
+        """
+        env = boto3.session.Session(profile_name=self.env, region_name=self.region)
+        elbclient = env.client('elb')
+
+        elb_settings = self.properties['elb']
+        log.debug('Block ELB Settings Pre Configure Load Balancer Attributes:\n%s', pformat(elb_settings))
+
+        for job in json.loads(json_data)['job']:
+            load_balancer_attributes = {    
+                'CrossZoneLoadBalancing': {
+                    'Enabled': True
+                 },
+                 'AccessLog': {
+                     'Enabled': False,
+                 },
+                 'ConnectionDraining': {
+                     'Enabled': False,
+                 },
+                 'ConnectionSettings': {
+                     'IdleTimeout': 60
+                 }
+            }
+            if elb_settings.get('connection_draining_timeout'):
+                connection_draining_timeout = int(elb_settings['connection_draining_timeout'])
+                log.info("Adding Backend Connection Draining of %d", connection_draining_timeout)
+                load_balancer_attributes['ConnectionDraining'] = {
+                        'Enabled': True,
+                        'Timeout': connection_draining_timeout
+                }
+            if elb_settings.get('idle_timeout'):
+                idle_timeout = int(elb_settings['idle_timeout'])
+                log.info("Adding ELB Idle Timeout of %d", idle_timeout)
+                load_balancer_attributes['ConnectionSettings'] = {
+                        'IdleTimeout': idle_timeout
+                }
+            if elb_settings.get('access_log'):
+                access_log_bucket_name = elb_settings['access_log']['bucket_name']
+                access_log_bucket_prefix = elb_settings['access_log']['bucket_prefix']
+                access_log_emit_interval = int(elb_settings['access_log']['emit_interval'])
```

Comment:
That is correct, but what is to stop someone from making the items out of order? This is explicit and no question on order.